src/app/common-ui/Popup/index.js

```diff
@@ -17,29 +26,24 @@ function Popup({ show, title, content, buttonText, variant, onButtonClick }) {
     }
   }, [show]);
 
+  const selectSign = (variant) => {
+    if (variant === POPUP_VARIANT.ERROR) return iconError;
+    if (variant === POPUP_VARIANT.SUCESS) return iconSuccess;
+    if (variant === POPUP_VARIANT.WARNING) return iconWarning;
+    return '';
+  };
+
   return !isPristine ? (
```

Comment:
The logic here is not very reliable.
I ignore this issue first, we will fix later.